### ISSUE TITLE
[pickers][DatePicker] Blocked `aria-hidden` on an element because its descendant retained focus

### ISSUE THREAD

**Issue by erelzohar** [mui/mui-x#16096]
### Related page

https://mui.com/x/react-date-pickers/accessibility/

### Kind of issue

Missing information

### Issue description

the option to disable aria-hidden is missing , like in the simple modal component
"closeAfterTransition" attribute

**Search keywords**: DatePicker mui aria-hidden

**Comment by noraleonte** [comment]
Hey there! 👋 Thanks for reporting this. I'm not sure I understand the problem.. Could you link a specific demo or name the component that you are having trouble with?

**Comment by erelzohar** [comment]
Hi ! thanks for the response ,
my problem is  basically the same problem that was mentioned here : 
https://stackoverflow.com/questions/79006592/aria-hidden-warning-on-closing-mui-dialogue

but it looks like the DatePicker component doesnt have the same props like the mui-dialogue ,
im looking for the prop - "closeAfterTransition" .
by the article above it should solve the conosle error "Blocked aria-hidden on an element because its descendant retained focus"

**Comment by michelengelen** [comment]
Hey @erelzohar I cannot reproduce this as well. Could you give passing the prop via slotProps a try?

```tsx
slotProps={{
  dialog: { closeAfterTransition: true }
}}
```

**Comment by erelzohar** [comment]
Hi thanks for your help 
you’re correct,  I’m using the responsive DatePicker and the error occurs only on the mobile version, I’m not able to screenRecord right now but I’ll give -
 “slotProps={{
  dialog: { closeAfterTransition: true }
}}”
a try later and if the error continues I’ll send a screen record with examples , if it works I’ll send you a hug🤗
